CaptchaBuilder.php

```diff
@@ -128,19 +133,15 @@ public function setInterpolation($interpolate = true)
      */
     public $tempDir = 'temp/';
 
-    public function __construct($phrase = null, PhraseBuilderInterface $builder = null)
```

Comment:
If create phrase in construct, we can not change the length except add a length param in construct.Create phrase in build() we can do both change the length(setPhraseLen)  and the phrase(setPhrase)